mobile/.bazelrc

```diff
@@ -144,11 +144,12 @@ build:coverage --instrumentation_filter="//library/common[/:]"
 # GITHUB_TOKEN auth) so we have to override it to 'false' here.
 build:remote-ci-common --config=remote
 build:remote-ci-common --auth_enabled=false
+build:remote-ci-common --remote_cache=grpcs://envoy.cluster.engflow.com
 build:remote-ci-common --remote_executor=grpcs://envoy.cluster.engflow.com
 build:remote-ci-common --bes_backend=grpcs://envoy.cluster.engflow.com/
 build:remote-ci-common --bes_results_url=https://envoy.cluster.engflow.com/invocation/
+build:remote-ci-common --experimental_credential_helper=%workspace%/bazel/engflow-bazel-credential-helper.sh
 build:remote-ci-common --bes_lifecycle_events
-build:remote-ci-common --bes_upload_mode=fully_async
```

Comment:
Why this change?

Reply:
Otherwise, Bazel won't wait for BES to be fully uploaded, which can lead to incomplete streams if using short-lived runners like GH actions because Bazel server will just get terminated.